package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "@pmedianetwork/eslint-config",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
please don't update the version now - we update the version when we do a release. See the release-management workflow here: https://github.com/pmedianetwork/adverity-components#release-management

Reply:
changed in [622450](https://github.com/pmedianetwork/eslint-config/pull/42/commits/622450ef9b634863dc4701fca365509a44fcf6b5)